app/mainwindow2.cpp

```diff
@@ -380,6 +381,13 @@ void MainWindow2::updateSaveState()
     setWindowModified( mEditor->currentBackup() != mBackupAtSave );
 }
 
+void MainWindow2::clearRecentFilesList()
+{
+    mRecentFileMenu->clear();
+    InformativeDialog *informDialog = new InformativeDialog("Notice!", "You have sucessfully cleared the list");
+    informDialog->exec();
```

Comment:
I came to the conclusion that ultimately I’m in favour of enabled/disabled, but it could be a good idea to toggle the text as well like some programs do, e.g. between “Clear” (enabled) and “Currently empty” (disabled) or something like that. That’d be clearer and at the same time it feels like one part of the program (the preferences dialog) is actually “communicating” with the other (the recent files list) rather than just showing a confirmation all the time no matter what. It would still be nice to have a third voice on this, though.